README.md

```diff
@@ -18,7 +18,7 @@ The build environment brings all the necessary parts required to build a Git for
 
 ## Git for Windows
 
-The most important part of Git for Windows is [Git](https://git-scm.com/), obviously. The Git for Windows project maintains [a friendly fork](https://github.com/git-for-windows/git) of the "upstream" [Git project](https://github.com/git/git). The idea is that the Git for Windows repository serves as a test bed to develop patches and patch series that are specific to the Windows port, and once the patches stabilized, they are [submitted upstream](https://github.com/git-for-windows/git/tree/master/Documentation/SubmittingPatches).
+The most important part of Git for Windows is [Git](https://git-scm.com/), obviously. The Git for Windows project maintains [a friendly fork](https://github.com/git-for-windows/git) of the "upstream" [Git project](https://github.com/git/git). The idea is that the Git for Windows repository serves as a test bed to develop patches and patch series that are specific to the Windows port, and once the patches stabilized, they are [submitted upstream](https://github.com/git-for-windows/git/tree/main/Documentation/SubmittingPatches).
```

Comment:
What's different about this line? CRLF?

Reply:
The branch name.

Reply:
Yep, for some reason, GitHub does not highlight it. The last URL has `master` in it, and I replaced it by `main`.

Reply:
ELINETOOLONG?

Reply:
It's an `.md`... so the lines are expected to be wrapped after rendering them.

Reply:
Yes, I know. I was giving a possible reason why the diff highlighting doesn't work :-)